msteams-platform/graph-api/proactive-bots-&-messages/graph-proactive-bots-&-messages.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use Microsoft Graph to enable proactive bot installation and messaging in Teams
+description: Describes proactive messaging in Teams and how to implement.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams proactive messaging chat installation Graph
+---
+# Enable proactive bot installation and messaging in Teams with Microsoft Graph(Private Preview)
```

Comment:
Space before the parenthesis.

I would explicitly use the phrase "proactive messaging" to help with SEO and recognition. So: "Enable proactive bot installation and **proactive** messaging in Teams with Microsoft Graph (Private Preview)".

Reply:
Also, I like your phrase "proactive bot installation", wish I'd thought of it :)